src/containers/landlord/listingInfo.jsx

```diff
@@ -1,88 +1,121 @@
-import React, {useState} from "react";
-import { EditText } from 'react-edit-text';
+import React, { useState } from "react";
+import { EditText } from "react-edit-text";
 import "./listingInfo.scss";
-import {useLocation} from "react-router-dom";
+import { useLocation } from "react-router-dom";
 
 const ListingInfo = () => {
+  const location = useLocation();
 
-    const location = useLocation();
+  const currentListing = location.state;
 
-    const currentListing = location.state;
+  const [editState, setEditState] = useState(true);
 
-    const [editState, setEditState] = useState(true);
-
-    const handleClick = () => {
-        if (editState === true) {
-            setEditState(false);
-            console.log("success");
-        }
-        else {
-            setEditState(true);
-            console.log("success2");
-            console.log(currentListing);
-        }
+  const handleClick = () => {
+    if (editState === true) {
+      setEditState(false);
+      console.log("success");
+    } else {
+      setEditState(true);
+      console.log("success2");
+      console.log(currentListing);
```

Comment:
can remove log statements

Reply:
This is coming from the main branch. I think @ssreepathy added this.